packages/@okta/vuepress-site/docs/guides/protect-your-api/before-you-begin/nodeexpress/create-app.md

```diff
@@ -0,0 +1,3 @@
+The [Express getting started guide](https://expressjs.com/en/starter/installing.html) teaches you how to create a new Express app.
+
+Or, if you want to skip this guide and just download a working sample app, download our [Node Express example](https://github.com/okta/samples-nodejs-express-4/tree/master/okta-hosted-login).
```

Comment:
Do we use informal terms like "sample app" in our docs, or should we expand it to "sample application"? @jakubvul